space/internal/components/aclnotifications/notifications.go

```diff
@@ -71,9 +75,13 @@ func (n *aclNotificationSender) Run(ctx context.Context) (err error) {
 }
 
 func (n *aclNotificationSender) Close(ctx context.Context) (err error) {
-	if err := n.batcher.Close(); err != nil {
```

Comment:
Why was this needed? It looks weird, we should wait when batcher closes synchronously